contains-duplicate/changchanghwang.go

```diff
@@ -0,0 +1,16 @@
+// Time complexity, O(n)
+// Space complexity, O(n)
+// 풀이
+// nums 배열을 순회하면서 hashMap에 num을 key로, 존재 여부를 value로 저장한다.
+// 만약 이미 존재하는 key라면 true를 반환하고, 순회를 전부 했는데도 중복이 없다면 false를 반환한다.
+func containsDuplicate(nums []int) bool {
+	hashMap := map[int]bool{}
+	for _, num := range nums {
+		if hashMap[num] {
+			return true
+		} else {
+			hashMap[num] = true
+		}
```

Comment:
```suggestion
		if hashMap[num] {
			return true
		} 
		hashMap[num] = true
```

리뷰를 해야한다는 강박에 굳이 한가지 suggestion 해보았습니다.. 🙏

Reply:
ㅋㅋㅋ 제안 감사합니다 ㅎㅎ 사실 쓸모는 없는 else문이라 suggestion 바로 적용했습니다 ㅋㅋㅋ